samples/deployment/aws/deploy_and_destroy.yaml

```diff
@@ -0,0 +1,130 @@
+- name: "My AWS Infra"
```

Comment:
Let's rephrase to `My AWS infrastructure`.

Reply:
fixed